data/schema/units/specials.cfg

```diff
@@ -19,6 +19,27 @@
 	{FILTER_TAG "filter_defender" unit {FILTER_TAG "filter_weapon" weapon ()}}
 	{WML_MERGE_KEYS}
 [/tag]
+# A few specials inheriting from ~generic~ are included here so that unit abilities can then inherit from them.
```

Comment:
This sounds good, but the corresponding changes to the abilities' validation seems to be missing.

Reply:
These were added because ability tags were already referencing them as super-tags, but they didn't exist, so that gave an error in the schema self-validator.